scormxblock/scormxblock.py

```diff
@@ -345,7 +344,10 @@ def get_sha1(self, file_descriptor):
         """
         block_size = 8 * 1024
         sha1 = hashlib.sha1()
-        for block in iter(partial(file_descriptor.read, block_size), ''):
```

Comment:
please clarify why do we need that change?

Reply:
Because in other way we got an infinite loop of iterations

Reply:
How it could be?